doc/conftest.py

```diff
@@ -129,6 +137,16 @@ def pytest_runtest_setup(item):
     elif fname.endswith("statistical_inference/unsupervised_learning.rst"):
         setup_unsupervised_learning()
 
+    rst_files_requiring_matplotlib = [
+        "modules/partial_dependence.rst",
+        "modules/tree.rst",
+        "tutorial/statistical_inference/settings.rst",
+        "tutorial/statistical_inference/supervised_learning.rst",
```

Comment:
The forward line slash would also work on Linux?

Reply:
Not sure but there are other examples like this a few lines above in this file ...